This removes -static compile option. The -static option tells GCC to
link this statically with the libc, which we do not want in OpenWrt. We
want to link everything dynamically to the libc. This fixes a compile
problem with glibc.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -198,10 +198,10 @@ AM_CFLAGS = -Wall -Wimplicit -Wunused -W
 
 if IFXOS_ENABLE
 AM_LDFLAGS= \
-	-Bstatic -dn -static @IFXOS_LIBRARY_PATH@
+	-Bstatic -dn @IFXOS_LIBRARY_PATH@
 else
 AM_LDFLAGS= \
-	-Bstatic -dn -static
+	-Bstatic -dn
 endif
 
 #
@@ -302,7 +302,7 @@ mei_cpe_appl_ldflags= $(ADD_APPL_LDFLAGS
 else
 if TARGET_ADM5120_MIPSEL
 mei_cpe_appl_cflags =  -O1 -g
-mei_cpe_appl_ldflags = -static
+mei_cpe_appl_ldflags =
 else
 mei_cpe_appl_cflags =  -DPPC
 endif
@@ -335,7 +335,7 @@ endif
 mei_cpe_drv_test_CFLAGS = $(mei_cpe_app_common_cflags) \
 				$(mei_cpe_appl_cflags) $(MEI_DRV_TARGET_OPTIONS)
 
-mei_cpe_drv_test_LDFLAGS = $(mei_cpe_appl_ldflags) -Bstatic -dn -static @IFXOS_LIBRARY_PATH@ \
+mei_cpe_drv_test_LDFLAGS = $(mei_cpe_appl_ldflags) -Bstatic -dn @IFXOS_LIBRARY_PATH@ \
 							$(dsl_cpe_mei_LDFLAGS)
 
 mei_cpe_drv_test_LDADD = -lifxos $(dsl_cpe_mei_LDADD)
@@ -356,7 +356,7 @@ endif
 
 mei_cpe_drv_dbg_strm_dmp_CFLAGS = $(mei_cpe_app_common_cflags) \
 				$(mei_cpe_appl_cflags) $(MEI_DRV_TARGET_OPTIONS)
-mei_cpe_drv_dbg_strm_dmp_LDFLAGS = $(mei_cpe_appl_ldflags) -Bstatic -dn -static @IFXOS_LIBRARY_PATH@ \
+mei_cpe_drv_dbg_strm_dmp_LDFLAGS = $(mei_cpe_appl_ldflags) -Bstatic -dn @IFXOS_LIBRARY_PATH@ \
 									$(dsl_cpe_mei_LDFLAGS)
 mei_cpe_drv_dbg_strm_dmp_LDADD = -lifxos $(dsl_cpe_mei_LDADD)
 
